packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Placeholder.Example.tsx

```diff
@@ -8,9 +8,9 @@ export class TextFieldPlaceholderExample extends React.Component<any, any> {
     return (
       <div className="docs-TextFieldExample">
         <TextField placeholder="I am a placeholder." ariaLabel="Please enter text here" />
-        <TextField disabled={true} placeholder="I am disabled. " />
-        <TextField required={true} placeholder="I am required." />
-        <TextField errorMessage="Error message" placeholder="I have an error message." />
+        <TextField disabled={true} placeholder="I am disabled. " ariaLabel="Please enter text here" />
```

Comment:
Should the ariaLabels reflect what the example is about, rather than all being the same? Also curious how the combo of ariaLabel and placeholder is read in the screen reader.

Reply:
I was taking the impression that the example should be how we expect the controls to be used, rather than document the examples themselves. I do take them all being the same as confusing though, I will amend the labels.

Reply:
This seems better, but you should make sure it reads reasonably (preferably not reading both the placeholder and the aria label)